### DISCUSSION TITLE
xmake 打包的时候无法带上头文件的目录？

### DISCUSSION THREAD

**Discussion by RimuruChan** [xmake-io/xmake#3500] [Q&A] [answered]
所有的头文件都堆在根目录会出现重名头问题，想问一下有没有什么办法改变这个打包的行为

**Comment by waruqi** [comment, marked as answer]
看下文档说明 https://xmake.io/#/zh-cn/manual/project_target?id=targetadd_headerfiles

可以保留目录结构的
